test/e2e/e2e_test.go

```diff
@@ -304,7 +312,7 @@ var _ = ginkgo.Describe("Kueue", func() {
 			spotRF = testing.MakeResourceFlavor("spot").
 				Label("instance-type", "spot").Obj()
 			gomega.Expect(k8sClient.Create(ctx, spotRF)).Should(gomega.Succeed())
-			clusterQueue = testing.MakeClusterQueue("cluster-queue").
+			clusterQueue = testing.MakeClusterQueue("cluster-queue").QueueingStrategy("StrictFIFO").
```

Comment:
```suggestion
			clusterQueue = testing.MakeClusterQueue("cluster-queue").
```

Please revert this change.

Reply:
Done